services/apps/premium/members_enrichment_worker/src/activities/enrichment.ts

```
@@ -198,6 +214,369 @@
+            updateLastEnrichedDate(tx.transaction(), memberId, existingMemberData.tenantId),
+          )
+  
+          await Promise.all(promises)
+        })
+  
+        */
+
+    svc.log.debug({ memberId }, 'Member sources processed successfully!')
+    return true
+
+    return false
```

Biome (1.9.4)

[error] This code is unreachable



(lint/correctness/noUnreachable)